src/Intervention/Image/Imagick/Decoder.php

```diff
@@ -98,6 +105,11 @@ public function initFromBinary($binary)
         $image = $this->initFromImagick($core);
         $image->mime = finfo_buffer(finfo_open(FILEINFO_MIME_TYPE), $binary);
 
+        // set standard IANA mime
+        if ($image->mime === 'image/x-icon') {
```

Comment:
I got different results on my machine to the Travis CI build here. The IANA standard MIME is `image/vnd.microsoft.icon`, so I've ensured that's what ends up in the `$img->mime` property if the non-standard `image/x-icon` is returned.